### PR TITLE
Support custom settings

### PR DESCRIPTION
**Pull request by winebarrel**
Fix to pass settings info when creating a server.

```ruby
server = GrpcKit::Server.new([
  [DS9::Settings::MAX_FRAME_SIZE, 65535]
])
```
@ganmacs Please review PR 🙏

### CODE REVIEW

File: lib/grpc_kit/server.rb [hunk 1]
```diff
@@ -19,6 +19,7 @@ def initialize(interceptors: [], shutdown_timeout: 30, min_pool_size: nil, max_p
       @rpc_descs = {}
       @mutex = Mutex.new
       @stopping = false
+      @settings = []
```
Comment (ganmacs) [inline lib/grpc_kit/server.rb]: `@settings = settings` ?

Comment (winebarrel) [reply lib/grpc_kit/server.rb]: 🙇  fixed a3b406f

Comment (ganmacs) [reply lib/grpc_kit/server.rb]: L22 should be fixed. not L18.

Comment (winebarrel) [reply lib/grpc_kit/server.rb]: 🙇  3db6541da38f6c28861c54efda7f829c755320a4

**Comment by winebarrel** [comment]
Thank you!!

**Pull request merged**